src/platform/plugins/shared/discover/public/application/main/state_management/redux/types.ts

```diff
@@ -45,12 +46,19 @@ export interface InternalStateDataRequestParams {
   searchSessionId: string | undefined;
 }
 
+interface TabGlobalState {
+  timeRange?: TimeRange;
+  refreshInterval?: RefreshInterval;
+  filters?: Filter[];
+}
+
 export interface TabState extends TabItem {
-  lastPersistedGlobalState: {
-    timeRange?: TimeRange;
-    refreshInterval?: RefreshInterval;
-    filters?: Filter[];
-  };
+  // Initial app and global state for the tab (provided before the tab is initialized).
+  initialAppState?: DiscoverAppState;
+  initialGlobalState?: TabGlobalState;
```

Comment:
Going to also use it to solve the following task where the query was wrong for unvisited tabs
- https://github.com/elastic/kibana/issues/221507